server/audit_math/bravo.py

```diff
@@ -185,21 +198,21 @@ def bravo_sample_sizes(
     # max here handles cases where, due to rounding error,
     # the base (content) of the radical is trivially
     # negative for p_completion very close to 0.5.
-    radical = math.sqrt(max(0, q_b ** 2 - 4 * q_a * q_c))
+    radical = math.sqrt((q_b ** 2 - 4 * q_a * q_c).max(0))
```

Comment:
why change `max` but not `math.sqrt`? in the decimal docs it seems like they use the builtin `max` but use the decimal's `sqrt` method (see examples a few paragraphs down in https://docs.python.org/3/library/decimal.html#quick-start-tutorial)

Reply:
in general i'm confused about when something needs to be a decimal and when it doesn't

Reply:
Ah, in this particular instance I suppose it doesn't matter. Generally, things need to be Decimal when an operation is being performed with another Decimal number, and/or we don't want to lose precision.

Here, `radical` needs to be a float because it is passed into some stats code that expects a float. I could cast it later on, but I found I was casting it a bunch of times later on.